<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>811</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_map">
    <property name="title">
     <string>Add Map...</string>
    </property>
    <addaction name="actionAddConstantMap"/>
    <addaction name="actionAddIdentityMap"/>
    <addaction name="actionAddAffineMap"/>
    <addaction name="actionAddPolynomialMap"/>
    <addaction name="actionAddFunctionMap"/>
    <addaction name="actionAddLuaMap"/>
    <addaction name="actionAddASAGI"/>
    <addaction name="actionAddSCECFile"/>
    <addaction name="actionAddEvalModel"/>
    <addaction name="actionAddOptimalStress"/>
    <addaction name="actionAddAndersonianStress"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Filter">
    <property name="title">
     <string>Add Filter...</string>
    </property>
    <addaction name="actionAddAny"/>
    <addaction name="actionAddAxisAlignedCuboidalDomainFilter"/>
    <addaction name="actionAddSphericalDomainFilter"/>
    <addaction name="actionAddGroupFilter"/>
    <addaction name="actionAddSwitch"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Builders">
    <property name="title">
     <string>Add Builders</string>
    </property>
    <addaction name="actionAddLayeredModel"/>
    <addaction name="actionAddInclude"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAdd_map"/>
   <addaction name="menuAdd_Filter"/>
   <addaction name="menuAdd_Builders"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionAddConstantMap">
   <property name="text">
    <string>Constant Map</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="user-trash">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAddAffineMap">
   <property name="text">
    <string>Affine Map</string>
   </property>
  </action>
  <action name="actionAddAny">
   <property name="text">
    <string>Any</string>
   </property>
  </action>
  <action name="actionAddIdentityMap">
   <property name="text">
    <string>Identity Map</string>
   </property>
  </action>
  <action name="actionAddAxisAlignedCuboidalDomainFilter">
   <property name="text">
    <string>Axis Aligned Cuboidal Domain Filter</string>
   </property>
   <property name="toolTip">
    <string>Axis Aligned Cuboidal Domain Filter</string>
   </property>
  </action>
  <action name="actionAddSphericalDomainFilter">
   <property name="text">
    <string>Spherical Domain Filter</string>
   </property>
  </action>
  <action name="actionAddGroupFilter">
   <property name="text">
    <string>Group Filter</string>
   </property>
  </action>
  <action name="actionAddSwitch">
   <property name="text">
    <string>Switch</string>
   </property>
  </action>
  <action name="actionAddPolynomialMap">
   <property name="text">
    <string>Polynomial Map</string>
   </property>
  </action>
  <action name="actionAddFunctionMap">
   <property name="text">
    <string>Function Map</string>
   </property>
  </action>
  <action name="actionAddASAGI">
   <property name="text">
    <string>ASAGI</string>
   </property>
  </action>
  <action name="actionAddSCECFile">
   <property name="text">
    <string>SCEC File</string>
   </property>
  </action>
  <action name="actionAddEvalModel">
   <property name="text">
    <string>Eval Model</string>
   </property>
  </action>
  <action name="actionAddOptimalStress">
   <property name="text">
    <string>Optimal Stress</string>
   </property>
  </action>
  <action name="actionAddAndersonianStress">
   <property name="text">
    <string>Andersonian Stress</string>
   </property>
  </action>
  <action name="actionAddInclude">
   <property name="text">
    <string>Include</string>
   </property>
  </action>
  <action name="actionAddLayeredModel">
   <property name="text">
    <string>Layered Model</string>
   </property>
  </action>
  <action name="actionAddLuaMap">
   <property name="text">
    <string>Lua Map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
